chia/wallet/did_wallet/did_wallet.py

```diff
@@ -605,10 +608,20 @@ async def transfer_did(self, puzzle_hash: bytes32, fee: uint64) -> TransactionRe
         assert coins is not None
         coin = coins.pop()
 
+        new_did_puz = did_wallet_puzzles.create_innerpuz(
+            new_puzhash,
+            0,
```

Comment:
I had a discussion about if we should transfer the recovery list to the new owners with Cameron. In most cases it should be some one transfer his DID to a new wallet, he may not want to set the recovery list again after the transfer.